src/main/webapp/index.html

```diff
@@ -12,6 +12,14 @@
   <body>
     <header id="header" class="base">
       <p id="title">Engage</p>
+      <p id="instructions">
+        We help you engage more deeply with your social media feed. 
+        Copy and paste a post, or upload an image from you feed. 
+        Then, press submit, and you'll see a collection of books and community projects
+        about each important keyword in your feed. If you'd like to contact a local politician
+        about an issue, enable the use of location data and find your local representatives 
+        on the right side of the screen.
```

Comment:
[Optional] I'd create a second paragraph starting with "If you'd like to contact..."